UML-Diagram/ReturnSystem.puml

```diff
@@ -0,0 +1,28 @@
+@startuml ReturnSystem
+
+package "Return System" {
+    interface IReturnStrategy {
+        + ReturnProcess(Book[] books)
+    }
+
+    class ReturnContext {
+        - borrow : Borrow
+        - fine : Fine
+        - returnStrategy : IReturnStrategy
+        + ReturnContext(IReturnStrategy returnStrategy)
+        + setReturnStrategy(IReturnStrategy returnStrategy) : void
+        + performReturn() : void
+    }
+
+    class RegularReturnStrategy implements IReturnStrategy {
+        + ReturnProcess(Book[] books)
```

Comment:
This two lists are not used in other classes, I think there is  two lists one for books that doesnt have a fine and the list of returned books that has a fine related, but the class Fine contains already that information, Book class doesnt know the fine context.